### PR TITLE
dialogue iteration 1 complete

### PR DESCRIPTION
**Pull request by dkaloger**
sorry for the messy history we had some git problems

### CODE REVIEW

File: UOP1_Project/Assets/Dialogue/scripts/dialoguetriger.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class dialoguetriger : MonoBehaviour
```
Comment (MatasV) [inline UOP1_Project/Assets/Dialogue/scripts/dialoguetriger.cs:5]: PascalCase would be preferable for the class name

File: UOP1_Project/Assets/Dialogue/scripts/conversation.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "Conversation", menuName = "ScriptableObjects/Conversation", order = 1)]
+public class conversation : ScriptableObject
```
Comment (MatasV) [inline UOP1_Project/Assets/Dialogue/scripts/conversation.cs:5]: PascalCase would be preferable for the class name

**Review by MatasV — CHANGES_REQUESTED** [review]
PascalCase pls

**Review by dkaloger — COMMENTED** [review]
tmp import made obsolete  and deleted

**Review by dkaloger — COMMENTED** [review]
old approach made obsolete everything deleted apart from DialogueRender.

**Review by dkaloger — COMMENTED** [review]
deleted will be added back on iteration 2

File: UOP1_Project/Assets/Dialogue/conversations/nice tree.asset [hunk 3]
```diff
@@ -23,4 +23,4 @@ MonoBehaviour:
     fontstyle: 3
     Speaking: {fileID: 10907, guid: 0000000000000000f000000000000000, type: 0}
     NameOveride: character2
-  triggered_once: 1
+  triggered_once: 0
```
Comment (MatasV) [inline UOP1_Project/Assets/Dialogue/conversations/nice tree.asset:26]: There might be problems here, remind me about this later on, @dkaloger

**Review by MatasV — APPROVED** [review]
Approved

File: UOP1_Project/Assets/Dialogue/conversations/nice tree.asset (hunk 3 above)
Comment (dkaloger) [reply UOP1_Project/Assets/Dialogue/conversations/nice tree.asset:26]: no it was done very intentionally . dialogs are ment to only be triggered once even after exiting and the rentering playmode

Comment (MatasV) [reply UOP1_Project/Assets/Dialogue/conversations/nice tree.asset:26]: Ok, then it's all good. Just wanted to make sure that the persistence is intended

**Comment by SoundGuy** [comment]
How do i check this out to maybe comment of do a code review?

**Comment by ManuelRauber** [comment]
@SoundGuy Code Reviews and commeting are via GitHub directly. Just click `Files Changed` and start commenting lines.

**Comment by SoundGuy** [comment]
Understanding the snapshot of a h its project through looking at code through github . Without the ability to look at things like prefabs or scenes doesn’t seem like a good approach. I can’t find a simple way to get a snapshot of the State of the projects asscociated with the PR to understand if there is anything one should look for or comments about.

**Comment by ciro-unity** [comment]
A bit simple in its approach, and some lines feel very hardcoded. Things like ` if (other.gameObject.name == "Pig" )` in the trigger for dialogues or `GameObject.FindGameObjectWithTag("subtitles").GetComponent<DialogueRendererer>().StartCoroutine(GameObject.FindGameObjectWithTag("subtitles").GetComponent<DialogueRendererer>().NewChat(Conversation)); ;` :D

Closing because we have other PRs that will probably be pulled. This said, don't be discouraged! @dkaloger @MatasV and @Aspekt1024, I've added you to the list of contributors nonetheless :)

@Aspekt1024 if you give me your Discord username, I'll give you the Open Project Contributor role there too. (the others already have it).

**Pull request closed without merging**